.github/workflows/main.yml

```diff
@@ -61,6 +61,27 @@ jobs:
       - name: Run test
         run: ${{ matrix.task }}
 
+  check-typos:
+    runs-on: ubuntu-latest
+    container: fedora:38  # CURRENT DEVELOPMENT ENVIRONMENT
+    steps:
+      - name: Install dependencies
+        run: >
+          dnf install -y
+          clang
+          glibc-static
+          make
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          components: cargo
+          toolchain: 1.71.1  # LOWEST SUPPORTED RUST TOOLCHAIN
```

Comment:
Use CURRENT instead of LOWEST. That's what I'll be using to run install typos, so that makes the most sense.

Reply:
OK; amended to use 1.72.0.